main.tf

```diff
@@ -3,125 +3,92 @@ locals {
+    postgres_port    = var.postgres_service.port
   }
 }
 
 data "template_file" "nomad_job_redash_scheduler" {
-  template = file("${path.module}/conf/nomad/redash_scheduler.hcl")
+  template = file("${path.module}/nomad/redash_scheduler.hcl")
   vars = {
-    # nomad
-    datacenters                   = local.datacenters
-    # redash
-    redash_scheduler_service_name = var.redash_scheduler_service_name
-    redash_image                  = var.redash_container_image
-    redash_mail_port              = var.redash_mail_port
-    redash_cpu                    = var.redash_resource_cpu
-    redash_memory                 = var.redash_resource_memory
-    redash_scheduler_count        = var.redash_scheduler_count
-    # postgres
-    postgres_service_name         = var.postgres_service_name
-    postgres_port                 = var.postgres_container_port
-    # redis
-    redis_service_name            = var.redis_service_name
-    redis_port                    = var.redis_container_port
-    # email
-    email_service_name            = var.email_service_name
-    email_port                    = var.email_container_port
+    datacenters  = local.datacenters
+    namespace    = var.nomad_namespace
+    image        = var.container_image
+    service      = var.service
+    host         = var.host
+    port         = var.port
+    cpu          = var.resource.cpu
+    memory       = var.resource.memory
+    cpu_proxy    = var.resource_proxy.cpu
+    memory_proxy = var.resource_proxy.memory
+    use_canary   = var.use_canary
+    # Redis
+    redis_service = var.redis_service.service
+    redis_port    = var.redis_service.port
+    # Postgres
+    postgres_service = var.postgres_service.service
+    postgres_port    = var.postgres_service.port
   }
 }
 
-data "template_file" "nomad_job_redis" {
-  template = file("${path.module}/conf/nomad/redis.hcl")
-  vars = {
-    # nomad
-    datacenters        = local.datacenters
-    # redis
-    redis_service_name = var.redis_service_name
-    redis_image        = var.redis_container_image
-    redis_port         = var.redis_container_port
-    redis_count        = var.redis_count
-  }
-}
 
-data "template_file" "nomad_job_email" {
-  template = file("${path.module}/conf/nomad/email.hcl")
-  vars = {
-    # nomad
-    datacenters        = local.datacenters
-    # email
-    email_service_name = var.email_service_name
-    email_image        = var.email_container_image
-    email_port         = var.email_container_port
-    email_count        = var.email_count
-  }
-}
+
+//data "template_file" "nomad_job_email" {
+//  template = file("${path.module}/conf/nomad/email.hcl")
+//  vars = {
+//    # nomad
+//    datacenters        = local.datacenters
+//    # email
+//    email_service_name = var.email_service_name
+//    email_image        = var.email_container_image
+//    email_port         = var.email_container_port
+//    email_count        = var.email_count
+//  }
+//}
```

Comment:
Er det noen grunn til å ta vare på det utkommenterte?

Reply:
Foreløpig har jeg ikke integrert mail-server, jeg er usikker på hva behovet er for det? Derfor har jeg bare kommentert det bort inntil det er avklart.